util/host/main.lua

```diff
@@ -1,22 +1,3 @@
-print("\n\n") --give us some room
-
-function usage()
-  print([[
-    
-    Moai Host Utility. Run from project root folder (outside lua source)
-    
-    Usage: moaiutil host <subcommand> <args>
-    
-    Subcommands:
-      host list - Lists available hosts
-      host init - Creates a template host config file used by subsequent commands
-      host build <hostname> - Creates (if it doesn't exist) in hosts folder, and (re)builds the host named <hostname>. 
-      host run <hostname> - Creates (if it doesn't exists) in hosts folder, (re)builds and runs the host named <hostname>. 
-      host create <hostname> - Creates the host in the hosts folder (removing old host) based on latest config settings.
-    ]])
```

Comment:
Okay, I just wanted to clean it up and not have copies of the text all over the place .. I could add an arg to usage() so if its nil, it spits out everything, if its "host", it spits out the host usage info, etc.